README.md

```diff
@@ -273,56 +301,28 @@ Android Studio and some older versions of other IntelliJ-based IDEs use a boot r
 To solve this issue, open the ["Choose Boot Runtime for the IDE"](https://www.jetbrains.com/help/idea/switching-boot-jdk.html) dialog where you can change the boot runtime to one that contains JCEF.
 
 # Troubleshooting
+
 The JFrog IntelliJ IDES Plugin uses the IntelliJ IDEA log files. By default, the log level used by the plugin is INFO.
 
 You have the option of increasing the log level to DEBUG. Here's how to do it:
 
 1. Go to **Help** | **Diagnostic Tools** | **Debug Log Settings...**
 2. Inside the **Custom Debug Log Configuration** window add the following line:
-```
+
+```java
 #com.jfrog.ide.idea.log.Logger
 ```
+
 To see the Intellij IDEA log file, depends on the IDE version and OS as described [here](https://intellij-support.jetbrains.com/hc/en-us/articles/207241085-Locating-IDE-log-files), go to **Help** | **Show/reveal Log in Explorer/finder/Konqueror/Nautilus**.
 
 # Reporting Issues
 
 Please report issues by opening an issue on [Github](https://github.com/jfrog/jfrog-idea-plugin/issues).
 
-# Contributions
-## Building and Testing the Sources
-
-To build the plugin sources, please follow these steps:
-1. Clone the code from git.
-2. Build and create the JFrog IDEA Plugin zip file by running the following gradle command.
-After the build finishes, you'll find the zip file in the *plugin/build/distributions* directory, located under the *jfrog-idea-plugin* directory.
-The zip file can be loaded into IntelliJ:
-    ```
-    gradle clean build
-    ```
-3. Python tests:
-    ```
-    gradle pythonTests
-    ```
-4. Integration tests:
-   * Make sure you have JFrog platform Instance with JAS enabled.
-   * Set the JFROG_IDE_PLATFORM_URL, JFROG_IDE_ACCESS_TOKEN environment variables with your JFrog platform URL, and access token.
-    Run the following command:
-     ```
-     gradle integrationTests
-     ```
-
-## Developing the Plugin Code
-If you'd like to help us develop and enhance the plugin, this section is for you.
-To build and run the plugin following your code changes, follow these steps:
-
-1. From IntelliJ, open the plugin project, by selecting *jfrog-idea-plugin/build.gradle* file.
-2. Build the sources and launch the plugin by the following these steps:
-* From the *Gradle Projects* window, expand *Idea --> Tasks -->  IntelliJ*
-* Run the *buildPlugin* task.
-* Run the *runIdea* task.
-
 ## Code Contributions
```

Comment:
Now that the contributions info was moved to a separate file, I suggest changing this section's name and make its title bigger (of course, if you agree with me, the title and the indentation in the table of contents need to be updated too):

"## Code Contributions" --> "# Contributions"